### DISCUSSION TITLE
Общее

### DISCUSSION THREAD

**Discussion by kot23rus** [kubcoder/KUBC.DAYZ.ITEMS#1] [General]
Общие вопросы и обсуждение мода

**Comment by Fom23** [comment]
Добрый день. планируется ли команда бессмертия, невидимости и свободной камеры?

**Reply by kot23rus** [reply]
Команда бессмертия существует и называется [god](https://kubcoder.ru/dayz/console/cmd/god). Невидимость и свободную камеру можно реализовать только используя клиентский мод. Данный мод позиционируется как серверный и не требующий установки на стороне клиента.

**Reply by Fom23** [reply]
> Команда бессмертия существует и называется [god](https://kubcoder.ru/dayz/console/cmd/god). Невидимость и свободную камеру можно реализовать только используя клиентский мод. Данный мод позиционируется как серверный и не требующий установки на стороне клиента.

где я тогда могу посмотреть полный список команд? внутри самого мода есть man?
как я понимаю тут не все команды `https://kubcoder.ru/dayz/items`

**Reply by kot23rus** [reply]
Смотри, мода с командами как минимум два
- [KUBC.DAYZ.CONSOLE](https://kubcoder.ru/dayz/console) - Серверный мод добавляющий глобальный чат и текстовые команды администрирования на сервер DAYZ.
- [KUBC.DAYZ.ITEMS](https://kubcoder.ru/dayz/items) Серверный мод который добавляет возможность администрирования игровых предметов

На странице каждого из модов можно найти полный список команд, а также примеры их использования.

**Comment by Fom23** [comment]
в логах сервера на Linux Debian 12

Log /home/steam/.local/share/DayZ/script_2026-02-28_16-55-37.log started at 28.02. 16:55:37

SCRIPT       : Registered 302 temporary action enum(s), UAN==302
SCRIPT       : ... Backlit Effects Enabled
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла Connect.json: File "$profile:KUBC\INFO\Connect.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Connect.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла PlayersCount.json: File "$profile:KUBC\INFO\PlayersCount.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayersCount.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла Suicide.json: File "$profile:KUBC\INFO\Suicide.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Suicide.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла PlayerDead.json: File "$profile:KUBC\INFO\PlayerDead.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerDead.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла Explosive.json: File "$profile:KUBC\INFO\Explosive.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Explosive.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла PlayerKill.json: File "$profile:KUBC\INFO\PlayerKill.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerKill.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла CarCrash.json: File "$profile:KUBC\INFO\CarCrash.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации CarCrash.json создаем новый
SCRIPT       : [Error][KUBC.INFO]Ошибка загрузки файла AiKill.json: File "$profile:KUBC\INFO\AiKill.json" does not exist, check the provided path
SCRIPT       : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации AiKill.json создаем новый
SCRIPT       : [Info][KUBC.CHAT]Файл настроек глобального чата не существует, создаем файл по умолчанию
SCRIPT       : [Info][KUBC.CMD]Добавлена команда [god]
SCRIPT       : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [god]
SCRIPT       : [Info][KUBC.CMD]Добавлена команда [heal]
SCRIPT       : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [heal]
SCRIPT       : [Info][KUBC.CMD]Добавлена команда [time]
SCRIPT       : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [time]


это так и должно быть?

**Reply by Fom23** [reply]
> в логах сервера на Linux Debian 12
> 
> Log /home/steam/.local/share/DayZ/script_2026-02-28_16-55-37.log started at 28.02. 16:55:37
> 
> SCRIPT : Registered 302 temporary action enum(s), UAN==302 SCRIPT : ... Backlit Effects Enabled SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Connect.json: File "$profile:KUBC\INFO\Connect.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Connect.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayersCount.json: File "$profile:KUBC\INFO\PlayersCount.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayersCount.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Suicide.json: File "$profile:KUBC\INFO\Suicide.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Suicide.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayerDead.json: File "$profile:KUBC\INFO\PlayerDead.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerDead.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Explosive.json: File "$profile:KUBC\INFO\Explosive.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Explosive.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayerKill.json: File "$profile:KUBC\INFO\PlayerKill.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerKill.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла CarCrash.json: File "$profile:KUBC\INFO\CarCrash.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации CarCrash.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла AiKill.json: File "$profile:KUBC\INFO\AiKill.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации AiKill.json создаем новый SCRIPT : [Info][KUBC.CHAT]Файл настроек глобального чата не существует, создаем файл по умолчанию SCRIPT : [Info][KUBC.CMD]Добавлена команда [god] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [god] SCRIPT : [Info][KUBC.CMD]Добавлена команда [heal] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [heal] SCRIPT : [Info][KUBC.CMD]Добавлена команда [time] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [time]
> 
> это так и должно быть?

я кажется понял... пути которые написаны в логах используют обратные слеши. если я правильно понимаю ваш мод можно использовать только под виндой? или я ошибаюсь?

**Reply by Fom23** [reply]
> > в логах сервера на Linux Debian 12
> > Log /home/steam/.local/share/DayZ/script_2026-02-28_16-55-37.log started at 28.02. 16:55:37
> > SCRIPT : Registered 302 temporary action enum(s), UAN==302 SCRIPT : ... Backlit Effects Enabled SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Connect.json: File "$profile:KUBC\INFO\Connect.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Connect.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayersCount.json: File "$profile:KUBC\INFO\PlayersCount.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayersCount.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Suicide.json: File "$profile:KUBC\INFO\Suicide.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Suicide.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayerDead.json: File "$profile:KUBC\INFO\PlayerDead.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerDead.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла Explosive.json: File "$profile:KUBC\INFO\Explosive.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации Explosive.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла PlayerKill.json: File "$profile:KUBC\INFO\PlayerKill.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации PlayerKill.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла CarCrash.json: File "$profile:KUBC\INFO\CarCrash.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации CarCrash.json создаем новый SCRIPT : [Error][KUBC.INFO]Ошибка загрузки файла AiKill.json: File "$profile:KUBC\INFO\AiKill.json" does not exist, check the provided path SCRIPT : [Warning][KUBC.INFO]Не смогли загрузить файл конфигурации AiKill.json создаем новый SCRIPT : [Info][KUBC.CHAT]Файл настроек глобального чата не существует, создаем файл по умолчанию SCRIPT : [Info][KUBC.CMD]Добавлена команда [god] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [god] SCRIPT : [Info][KUBC.CMD]Добавлена команда [heal] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [heal] SCRIPT : [Info][KUBC.CMD]Добавлена команда [time] SCRIPT : [Info][KUBC.CMD]Создаем файл настройки доступа по умолчанию для команды [time]
> > это так и должно быть?
> 
> я кажется понял... пути которые написаны в логах используют обратные слеши. если я правильно понимаю ваш мод можно использовать только под виндой? или я ошибаюсь?

dayz-standalone:~# cd $profile:KUBC
-bash: cd: :KUBC: Нет такого файла или каталога

**Reply by kot23rus** [reply]
видно под линукс не работает... видно при формировании пути... мы используем стиль Windows и у нас проблем нет, а на линуксе все через пятую точку, как следствие нифига не работает...

**Reply by kot23rus** [reply]
Надо будет подумать над версией для Linux... @Fom23 ты готов быть тестером?

**Comment by kot23rus** [comment]
@Fom23 Как ты подключаешь серверные моды под Linux??? второй день не могу победить эту хрень.... вроде уже исправил мод, но чет при подключении к серверу Linux все падает (на Win такой проблемы нет)
